types/variables/generator/variablesmap.go.tmpl

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// Code generated by types/variables/generator DO NOT EDIT.
+
+package variables
+
+func init() {
```

Comment:
Needs to happen on init because otherwise https://github.com/corazawaf/coraza/pull/641/files#diff-3a381248ac4252fe3790b68cb50658b876eb88b6eb07031df0c13844bc812bf7R238 complains about rulemap missing.